.buildkite/pipeline.yml

```diff
@@ -37,12 +37,12 @@ steps:
           provider: "gcp"
           image: "${GO_WINDOWS_AGENT_IMAGE}"
         artifact_paths:
-          - build/test-results/test-windows-*.xml
+          - build/test-results/*.xml
 
       - label: ":junit: Junit annotate"
         plugins:
           - junit-annotate#v2.4.1:
-              artifacts: "build/test-results/test-*.xml"
+              artifacts: "bbuild/test-results/*.xml"
```

Comment:
was the double "b" added on purpose here?

Reply:
no, by mistake.
I already got the error in the last stage ))
fixed.